recipes/pycrostates/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "pycrostates" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycrostates-{{ version }}.tar.gz
+  sha256: 8a9292d5f7cae08ac7aaf492460577305fb41ae05a08d0f79f5aaabbae411a43
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools >=61.0.0
+    - wheel
+  run:
+    - decorator
+    - importlib-metadata
+    - importlib-resources
+    - jinja2
+    - joblib
+    - matplotlib-base
+    - mne >=1.0.0
```

```suggestion
    - mne-base >=1.0.0
```

The issue is definitely coming from `mffpy` dependencies.. and we don't need it.
So let's pin to `mne-base` instead and simplify the installation since we anyway pin the extra mne dependencies that we are using: `joblib`, `scikit-learn`; in our req.